tests/wallet/rpc/test_dl_wallet_rpc.py

```diff
@@ -222,3 +224,57 @@ async def is_singleton_generation(rpc_client: WalletRpcClient, lid: bytes32, gen
             client_2.close()
             await client.await_closed()
             await client_2.await_closed()
+
+    @pytest.mark.limit_consensus_modes(allowed=[ConsensusMode.PLAIN, ConsensusMode.HARD_FORK_2_0], reason="save time")
+    @pytest.mark.parametrize("trusted", [True, False])
+    @pytest.mark.anyio
+    async def test_wallet_dl_verify_proof(
+        self, one_wallet_and_one_simulator_services: SimulatorsAndWalletsServices, trusted: bool, self_hostname: str
+    ) -> None:
+        [full_node_service], [wallet_service], bt = one_wallet_and_one_simulator_services
+        full_node_api = full_node_service._api
+        full_node_server = full_node_api.full_node.server
+        wallet_node = wallet_service._node
+
+        # Create fake proof
+        fakeproof = HashOnlyProof(
+            key_clvm_hash=std_hash(b"\1" + b"key"),
+            value_clvm_hash=std_hash(b"\1" + b"value"),
```

Comment:
See above about a classmethod.  Regardless, this is still manually applying the hash pattern.

Reply:
I did this specifically because I expect a user of this API set may have to "manually" verify hashes using plain SHA256 without the benefit of treehash and I wanted to test that you can verify things with just using SHA256 as long as you know the secret sauce of prepending 1

Reply:
Thanks.  Maybe add an abbreviated version of that explanation as a comment?

Reply:
Oh, actually here this is just fake data - ok I misspoke, this could do whatever